README.md

```diff
@@ -1 +0,0 @@
-# foo-corp-example
```

Comment:
Move the whole thing under a foo-corp subfolder

Reply:
done